### DISCUSSION TITLE
Way to validate `fields`?

### DISCUSSION THREAD

**Discussion by ramiroaraujo** [johnlindquist/kit#1197] [Q&A] [answered]
I don't think there's, right?
I may try to add something? something similar to the `arg` config, a `validate`  key that expects a function and return `boolean | string | Promise<boolean | string>`, etc. Sounds good?

**Comment by johnlindquist** [comment, marked as answer]
This should work:

```js
let [password] = await fields([
  {
    label: "Password",
    pattern: "^S.*t$", // Start with S, end with t, so "Script Kit" should be valid
    required: "true",
  },
])
```
